deployment/hasura/metadata/actions.graphql

```diff
@@ -234,13 +234,14 @@ type TypescriptFile {
   content: String
 }
 
-type CommandDictionaryResponse {
+type DictionaryResponse {
   id: Int!
-  command_types_typescript_path: String!
+  path: String!
   mission: String!
   version: String!
   parsed_json: CommandDictionary!
   created_at: String!
+  type : String!
 }
```

Comment:
`getCommandTypeScript` We can leave it as this because this file is generated by the command dictionary.

I will probably remove`getUserSequenceSeqJson` `GetUserSequenceSeqJsonBulkInput` as you can use hasura to fetch the user seqjson